common/src/configuration/utils.rs

```diff
@@ -245,7 +242,11 @@ pub fn default_config(bootstrap: &ConfigBootstrap) -> Config {
 
     cfg.set_default("wallet.base_node_service_peers", Vec::<String>::new())
         .unwrap();
-
+    cfg.set_default(
```

Comment:
?

Reply:
Was removed in PR#3600, a new wallet would fail when trying to create it. Included both fixes, this one and `.unwrap_or_else(Multiaddr::empty)`.  One can be removed depending on which we would want to keep.

Reply:
This was due to a bug introduced in #3600 but I don't think we need to add this default as it is a deprecated setting. In #3620 I provided an alternate fix to this issue so I think this can come out.

Reply:
Will remove it.